README.md

```diff
@@ -3,6 +3,18 @@
 [![Build Status](https://travis-ci.org/aerogear/digger-build-cli.png)](https://travis-ci.org/aerogear/digger-build-cli)
 [![License](https://img.shields.io/:license-Apache2-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 
+## Project Info
+
+|                 | Project Info  |
+| --------------- | ------------- |
+| License:        | Apache License, Version 2.0  |
+| Build:          | Python  |
+| Documentation:  | tbd  |
```

Comment:
Should we use [http://digger-build-cli.readthedocs.io/en/latest/](http://digger-build-cli.readthedocs.io/en/latest/) here?